evaluation.md

```diff
@@ -29,6 +33,7 @@
   - A minimum of 3 wireframes and/or mock-ups
 - Your repository includes a link to the deployed version of your project in the repository `URL` field
 - The repository includes a `readme.md` file containing:
+  - A description of the project
```

Comment:
:+1: It might be worth finding exceptional readmes from previous classes as an example